scripts/generate-connections.js

```diff
@@ -0,0 +1,72 @@
+const { default: createTestEnvironments } = require('./../src');
+const { v4 } = require('uuid');
+const path = require('path');
+const fs = require('fs/promises');
+
+const FILE_PATH = path.resolve(__dirname, '..', 'compass-connections.json');
```

Comment:
Small nit: we probably want to write this in the same folder where you ar running this from (or from an argument with this being a default, but this complicates things too much)

```suggestion
const FILE_PATH = path.resolve(process.cwd(), 'compass-connections.json');
```

Reply:
missed this one. will add it.

Reply:
added in https://github.com/mongodb-js/devtools-docker-test-envs/pull/27